app/src/app_config.py

```diff
@@ -3,3 +3,4 @@
 
 class AppConfig(PydanticBaseEnvConfig):
     embedding_model: str = "multi-qa-mpnet-base-dot-v1"
+    global_password: str | None = None
```

Comment:
PydanticBaseEnvConfig  ensures these defaults are overridden by any environment variables that are set